src/parsec.rs

```diff
@@ -55,6 +59,13 @@ const PARSEC_SIZE_LIMIT: u64 = 20_000_000;
 #[cfg(feature = "mock")]
 const PARSEC_SIZE_LIMIT: u64 = 500;
 
+/// Period within which the number of sent gossip messages is limited. When the period ends, the
+/// limit resets at a new period starts.
+pub const GOSSIP_PERIOD: Duration = Duration::from_secs(1);
+
+// Maximum number of gossip messages a node can send within one gossip period.
+const GOSSIP_LIMIT: usize = 5;
```

Comment:
It is dramatic, because the original values were rather arbitrary. The new values are based on the results of testnet benchmarks. More details here: https://hackmd.io/1tLHLeOkQtaRTaSwbt0glQ